README.md

```diff
@@ -71,6 +91,28 @@ Once you join, at the terminal you should see blocks produced quickly. Wait for
 2021-05-21T20:22:11Z INF committed state app_hash=6FD1FA7ACD8ACDAE027034ACCE8634BEC570E9839B6F9A7C6DC2011BD61780CF height=229885 module=state num_txs
 ...
 ```
+ Wait for your node to catch up with the network before proceeding. This can take a while. 
+ You can check the sync status by running:
+ ```shell script
+curl localhost:26657/status | jq '.result.sync_info'
```

Comment:
installing jq will come handy here for json processing, if not install this will throw error, maybe we can add `sudo apt-get install jq` just before the curl and remove from above ? what do you think?

Reply:
I think adding a quick note about installing jq via apt-get if they dont have it, would be great

Reply:
Sure